react-forms/src/DatePicker.tsx

```diff
@@ -80,28 +81,13 @@ export default function DateTimePicker<T>(props: IProps<T>) {
     React.useEffect(() => {
         if (!recordChange.current) return;
         const date = moment(boxRecord, boxFormat);
-        const validStartDate = moment("1753-01-01", "YYYY-MM-DD");
 
-        let valid = true;
+        validateDate(date);
```

Comment:
This returns a `boolean` but it is not used. This also causes issues because even if `validateDate` returns `false` here we still go into the `if` in Line 90

Reply:
The purpose of removing the valid was so their new date selection still gets saved. Previously we tracked if it was invalid and prevented from saving by not entering the if on like 90. As a result, we would have the user see the error from the component with their invalid date, but in realty the code would still have the old date saved. This would cause System Center to not realize the change in date. 

ie. if a valid date is after 2020 and originally the date is set to 2021
Then if they change it to 2019 the component would display 2019 and show an error for 2019, but System Center still believes it is 2021, due to not entering the if on like 90. 

validateDate returns a bool in case someone in the future wants to use the function.